core/dbt/adapters/sql/connections.py

```diff
@@ -131,14 +131,6 @@ def get_result_from_cursor(cls, cursor: Any, limit: Optional[int]) -> agate.Tabl
 
         return dbt.clients.agate_helper.table_from_data_flat(data, column_names)
 
-    @classmethod
-    def data_type_code_to_name(cls, type_code: Union[int, str]) -> str:
```

Comment:
This is already an abstract method. Move it up to ConnectionManager since BaseAdapter expects it to exist for some methods.